### PR TITLE
Add grpc-gateway for metrics service.

### PR DESCRIPTION
**Pull request by songy23**
Fixes https://github.com/census-instrumentation/opencensus-proto/issues/204.

### CODE REVIEW

File: src/mkgogen.sh [hunk 1]
```diff
@@ -29,3 +29,5 @@ OPENAPI_OUTDIR=../gen-openapi
 mkdir -p $OPENAPI_OUTDIR
 protoc --swagger_out=logtostderr=true,grpc_api_configuration=./opencensus/proto/agent/trace/v1/trace_service_http.yaml:$OPENAPI_OUTDIR \
   opencensus/proto/agent/trace/v1/trace_service.proto
+protoc --swagger_out=logtostderr=true,grpc_api_configuration=./opencensus/proto/agent/metrics/v1/metrics_service_http.yaml:$OPENAPI_OUTDIR \
```
Comment (odeke-em) [inline src/mkgogen.sh]: Did we miss a spot on Line 25?

Comment (songy23) [reply src/mkgogen.sh]: Good catch - added it.

**Comment by songy23** [comment]
Travis CI failure is not related to this change:
```bash
ERROR: /home/travis/.cache/bazel/_bazel_travis/6ad1f67d610d9e7d574100e0c95311d1/external/grpc_java/java_grpc_library.bzl:40:17: Traceback (most recent call last):
	File "/home/travis/.cache/bazel/_bazel_travis/6ad1f67d610d9e7d574100e0c95311d1/external/grpc_java/java_grpc_library.bzl", line 38
		rule(attrs = {"srcs": attr.label_list...")}, <2 more arguments>)
	File "/home/travis/.cache/bazel/_bazel_travis/6ad1f67d610d9e7d574100e0c95311d1/external/grpc_java/java_grpc_library.bzl", line 40, in rule
		attr.label_list(mandatory = True, non_empty = True, ..."])
'non_empty' is no longer supported. use allow_empty instead. You can use --incompatible_disable_deprecated_attr_params=false to temporarily disable this check.
```

Will fix in a separate PR.

**Comment by bogdandrutu** [comment]
Please rebase.

**Pull request merged**